translations/monero_it.ts

```diff
@@ -538,12 +538,12 @@
     <message>
         <location filename="../src/simplewallet/simplewallet.cpp" line="728"/>
         <source>Check tx proof for payment going to &lt;address&gt; in &lt;txid&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>Verifica tx proof per pagamento inviato a &lt;indirizzo&gt; in &lt;txid&gt;</translation>
```

Comment:
"Proof" should be translated with "prova"
`Verifica prova tx per pagamento inviato[...]`